producer.py

```diff
@@ -36,43 +40,29 @@
 
 from thoth.common import __version__ as __common__version__
 from thoth.storages import __version__ as __storages__version__
-from thoth.python import __version__ as __python__version__
+from thoth.messaging import __version__ as __messaging__version__
+from version import __version__
 
-__version__ = "0.6.1"
-__service_version__ = f"{__version__}+\
-    storages.{__storages__version__}.\
-        common.{__common__version__}.\
-            python.{__python__version__}"
+from thoth.messaging import MessageBase
+from thoth.messaging.package_releases import PackageReleaseMessage
 
 init_logging()
+app = MessageBase().app
```

Comment:
@KPostOffice looks like this is not used anywhere or is it the only place to use https://github.com/thoth-station/package-releases-job/pull/449/files#diff-4252fdf2230c8979fc9fbb96ab9bddb7R231?

Reply:
Yes it is required for the CLI :)